status.c

```diff
@@ -809,14 +809,23 @@ status_prompt_translate_key(struct client *c, key_code key, key_code *new_key)
 {
 	if (c->prompt_mode == PROMPT_ENTRY) {
 		switch (key) {
+		case '\001': /* C-a */
 		case '\003': /* C-c */
+		case '\005': /* C-e */
 		case '\007': /* C-g */
 		case '\010': /* C-h */
 		case '\011': /* Tab */
+		case '\013': /* C-k */
+		case '\016': /* C-n */
+		case '\020': /* C-p */
+		case '\024': /* C-t */
 		case '\025': /* C-u */
 		case '\027': /* C-w */
+		case '\031': /* C-y */
 		case '\n':
 		case '\r':
+		case KEYC_LEFT|KEYC_CTRL:
+		case KEYC_RIGHT|KEYC_CTRL:
```

Comment:
I see ~no harm~benefit in supporting these emacs control keys in vi normal mode. Any hot takes?